### PR TITLE
Change ConvertAs2_0 to ConvertAs2_1

### PR DESCRIPTION
**Pull request by ajeddeloh**
See https://github.com/coreos/bugs/issues/2171

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func Parse(data []byte) (types.Config, astnode.AstNode, report.Report) {
 // ConvertAs2_0 also accepts a platform string, which can either be one of the
```
Comment (ajeddeloh) [inline config/config.go]: Whoops, fixing this

**Comment by ajeddeloh** [comment]
Missed a lot of thing, not done

**Comment by ajeddeloh** [comment]
Ok, should have got them all now.

**Comment by sdemos** [comment]
What is the point of renaming these functions? It seems like having the version numbers in the function names has no tangible benefits that I can see. Why shouldn't we just remove them and then never have to deal with this search and replace?

**Comment by ajeddeloh** [comment]
I'd be fine with that. I think originally it wasn't clear if we'd support multiple output versions, so it was versioned, then when we bumped to 2.1.0, the function names were missed, leading to this PR.

@dgonyeo do you have a opinion on the matter?

**Comment by sdemos** [comment]
I would personally be in favor of ripping the band-aid off and just dropping the version numbers altogether.

**Comment by dghubble** [comment]
I assumed you'd be providing conversions to multiple Ignition spec versions, but I don't think that's the case. If the intent is to always produce a recent version (e.g. Ignition 2.1.0), I agree with @sdemos about dropping the numbers - vendoring the ct package provides the pinning.

**Comment by dgonyeo** [comment]
I'm in favor of dropping the version numbers.

Unless we want to revisit the decision of ct only producing a recent version (as opposed to being able to produce multiple versions), there's no reason for these names to be tied to the Ignition schema version number.

**Comment by sdemos** [comment]
@ajeddeloh do you have time to do that with this pr or should I take care of it and open a new one?

**Comment by ajeddeloh** [comment]
I'm working on a grub/fat/something in very early boot problem right now, and wont have time for this until that's done (timeboxed on this friday). If you want to pick it up, feel free.

**Comment by sdemos** [comment]
work to drop version numbers is in #127. closing this pr.

**Pull request closed without merging**